api/server/sdk/bucket_ops.go

```diff
@@ -0,0 +1,92 @@
+/*
+Package sdk is the gRPC implementation of the SDK gRPC server
+Copyright 2019 Portworx
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package sdk
+
+import (
+	"context"
+
+	"github.com/libopenstorage/openstorage/api"
+	bucket "github.com/libopenstorage/openstorage/bucket"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+// BucketServer is an implementation of the gRPC OpenStorageBucket interface
+type BucketServer struct {
+	server serverAccessor
+}
+
+func (s *BucketServer) driver(ctx context.Context) bucket.BucketDriver {
+	return s.server.bucketDriver(ctx)
+}
+
+//  Creates a new bucket
+func (s *BucketServer) Create(
+	ctx context.Context,
+	req *api.BucketCreateRequest,
+) (*api.BucketCreateResponse, error) {
+	if s.driver(ctx) == nil {
+		return nil, status.Error(codes.Unavailable, "Resource has not been initialized")
+	}
+	logrus.Info("bucket_driver.Fake delete bucket received")
+	name := req.GetName()
+
+	if len(name) == 0 {
+		return nil, status.Error(
+			codes.InvalidArgument,
+			"Must supply a unique name")
+	}
+	// Create bucket
+	id, err := s.driver(ctx).CreateBucket(name)
+
```

Comment:
nit: remove white space here. In Go, the err ~! nil is usually directly after the call, to avoid people unintentionally adding code that might reference a return variable.